### PR TITLE
FuelPHP Security Issues.

### PR DESCRIPTION
**Pull request by ularanigu**

### CODE REVIEW

**Comment by fabpot** [comment]
Thank you

**Comment by stof** [comment]
file names should be updated to match the convention of the repo (YYYY-MM-DD for dates)

File: fuel/core/29-06-2016-1.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     ImageMagick driver does not escape all shell arguments.
+link:      https://fuelphp.com/security-advisories
+cve:       ~
+branches:
+    master:
+        time:     2016-09-27 08:06:00
+        versions: ['<=1.8.0']
```
Comment (stof) [inline fuel/core/29-06-2016-1.yaml]: should be `>=1.1, <1.8.0.4`

File: fuel/core/14-04-2018-1.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title:     Crypt encryption compromised.
+link:      https://fuelphp.com/security-advisories
+cve:       ~
+branches:
+    master:
+        time:     2018-04-16 17:23:00
+        versions: ['<=1.8.0']
```
Comment (stof) [inline fuel/core/14-04-2018-1.yaml]: should be `<1.8.1`. The 1.8.0.4 release is also affected

**Pull request merged**